Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -949,12 +950,9 @@ class BrowserViewController: UIViewController {
 
     // MARK: Opening New Tabs
     func switchToPrivacyMode(isPrivate: Bool) {
-        let tabTrayController = self.tabTrayController ?? TabTrayController(tabManager: tabManager, profile: profile, tabTrayDelegate: self)
```

Comment:
Is it ok to not fallback to instantiating a new `TabTrayController` here?

Reply:
We dont really need a TabTrayController. On the iPad we were actually creating a TabTray and destroying it every time we created a new tab. I think initial we had the assumption that the TabTray is always around (hence the force unwrap on the property) but there is no need to always have a TabTray around.